### ISSUE TITLE
%matplotlib with no arg errors out

### ISSUE THREAD

**Issue by brunobeltran** [ipython/ipython#11819]
As in title, running ipython 7.6.1 and matplotlib 3.1.1, I get the following error (notice in particular that it only errors when no argument is passed).

```
[bbeltr1@tower0:~/developer/multi_locus_analysis] [base] master(+15/-19)+* 5m34s
± ipython
Python 3.7.3 (default, Mar 27 2019, 22:11:17)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.6.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: %matplotlib
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-a49a4afc47c5> in <module>
----> 1 get_ipython().run_line_magic('matplotlib', '')

~/.miniconda3/lib/python3.7/site-packages/IPython/core/interactiveshell.py in run_line_magic(self, magic_name, line, _stack_depth)
   2311                 kwargs['local_ns'] = sys._getframe(stack_depth).f_locals
   2312             with self.builtin_trap:
-> 2313                 result = fn(*args, **kwargs)
   2314             return result
   2315

</home/users/bbeltr1/.miniconda3/lib/python3.7/site-packages/decorator.py:decorator-gen-108> in matplotlib(self, line)

~/.miniconda3/lib/python3.7/site-packages/IPython/core/magic.py in <lambda>(f, *a, **k)
    185     # but it's overkill for just that one bit of state.
    186     def magic_deco(arg):
--> 187         call = lambda f, *a, **k: f(*a, **k)
    188
    189         if callable(arg):

~/.miniconda3/lib/python3.7/site-packages/IPython/core/magics/pylab.py in matplotlib(self, line)
     97             print(Available matplotlib backends: %s % backends_list)
     98         else:
---> 99             gui, backend = self.shell.enable_matplotlib(args.gui.lower())
    100             self._show_matplotlib_backend(args.gui, backend)
    101

AttributeError: 'NoneType' object has no attribute 'lower'

In [2]: %matplotlib qt5

In [3]: 
```

I'm inferring from a little `%debug`ing that this is due to "args.gui" being `None` because no "args" are passed to the `%matplotlib` magic. 

I would chase this farther but I have a deadline coming up this afternoon.

**Comment by Carreau** [comment]
sorry about that, I think the fix was #11814 I'll try to release soon.

**Comment by brunobeltran** [comment]
Thanks for the quick reply, that's the one!
